<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuringWindow</class>
 <widget class="QMainWindow" name="MeasuringWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Take picture and mark lines</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Distance from the camera:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="horizontalSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Measure!</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Aproximate height of last measured object:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="measuredValue">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Copy value to clipboard</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
